botocore/handlers.py

```diff
@@ -556,8 +562,13 @@ def change_get_to_post(request, **kwargs):
      AutoPopulatedParam('accountId', 'Note: this parameter is set to "-" by \
                          default if no value is not specified.')
      .document_auto_populated_param),
-    ('docs.*.glacier.*.complete-section',
+    ('docs.*.glacier.UploadArchive.complete-section',
+     AutoPopulatedParam('checksum').document_auto_populated_param),
+    ('docs.*.glacier.UploadMultipartPart.complete-section',
      AutoPopulatedParam('checksum').document_auto_populated_param),
+    ('docs.request-params.glacier.CompleteMultipartUpload.complete-section',
+     document_glacier_tree_hash_checksum()),
+
     # UserData base64 encoding documentation customizations
     ('docs.*.ec2.RunInstances.complete-section', document_base64_encoding()),
```

Comment:
I just happened to notice the same pattern of calling the function here as well when registering it. We should not be calling these.